base/process.jl

```diff
@@ -701,9 +701,30 @@ with a code of 0). An exception is raised if the process cannot be started.
 """
 success(cmd::AbstractCmd) = success(_spawn(cmd))
 
+struct ProcessExitedException
+    procs::Union{Vector{Process},Nothing}
+    # ProcessExitedException(nothing) is allowed for Distributed stdlib compat
+end
+ProcessExitedException() = ProcessExitedException(nothing)
+ProcessExitedException(proc::Process) = ProcessExitedException([proc])
+
+function show(io::IO, err::ProcessExitedException)
+    if procs == nothing
```

Comment:
`procs` isn't defined. Also it's better to use `===` when comparing against `nothing`.